docs/conf.py

```diff
@@ -26,6 +26,16 @@
 
 master_doc = 'docs/index'
 
+# Versions to show in the version menu
+# NOTE: we could be smarter and detect these from branch/tag names,
+# for example
+versions = [
+        'master',
+        'v0.4'
+        ]
```

Comment:
Please, could we avoid hardcoding this ? Instead let's runtime-detect all the versions we want to show documentation for and then, for instance, pass that as a comma-separated string in an suitably named environment variable, splitting+stripping it here into versions.